adapters/repos/db/vector/cache/sharded_lock_cache.go

```diff
@@ -42,13 +41,26 @@ type shardedLockCache struct {
 
 var shardFactor = uint64(512)
 
-const defaultDeletionInterval = 3 * time.Second
+const (
+	initialSize             = 1000
+	minimumIndexGrowthDelta = 2000
```

Comment:
@aliszka these values are also in hnsw maintenance.go. Ideally we store them once.

Reply:
We have addressed this by defining a unique set of constants.